Doc/library/_thread.rst

```diff
@@ -61,15 +61,27 @@ This module defines the following constants and functions:
       :func:`sys.unraisablehook` is now used to handle unhandled exceptions.
 
 
-.. function:: interrupt_main()
+.. function:: interrupt_main([signum])
```

Comment:
With the PEP 570 being implemented in Python 3.8, I suggest to use its syntax:

```suggestion
.. function:: interrupt_main(signum=signal.SIGINT, /)
```

Same remark for the docstring.

Reply:
Uh, will do.